src/routes/(console)/organization-[organization]/billing/planSummaryOld.svelte

```diff
@@ -0,0 +1,250 @@
+<script lang="ts">
+    import { base } from '$app/paths';
+    import { CardGrid } from '$lib/components';
+    import { Button } from '$lib/elements/forms';
+    import { toLocaleDate } from '$lib/helpers/date';
+    import { plansInfo, upgradeURL } from '$lib/stores/billing';
+    import { organization } from '$lib/stores/organization';
+    import type { Aggregation, Invoice, Plan } from '$lib/sdk/billing';
+    import { abbreviateNumber, formatCurrency, formatNumberWithCommas } from '$lib/helpers/numbers';
+    import { BillingPlan } from '$lib/constants';
+    import { Click, trackEvent } from '$lib/actions/analytics';
+    import {
+        Accordion,
+        Card,
+        Divider,
+        Icon,
+        Layout,
+        Tooltip,
+        Typography
+    } from '@appwrite.io/pink-svelte';
+    import { IconInfo, IconTag } from '@appwrite.io/pink-icons-svelte';
+    import CancelDowngradeModel from './cancelDowngradeModal.svelte';
+
+    export let currentPlan: Plan;
+    export let currentInvoice: Invoice | undefined = undefined;
+    export let availableCredit: number | undefined = undefined;
+    export let currentAggregation: Aggregation | undefined = undefined;
+
+    let showCancel: boolean = false;
+
+    const today = new Date();
+    const isTrial =
+        new Date($organization?.billingStartDate).getTime() - today.getTime() > 0 &&
+        $plansInfo.get($organization.billingPlan)?.trialDays;
+    const extraUsage = currentInvoice ? currentInvoice.amount - currentPlan?.price : 0;
+</script>
+
+{#if $organization}
+    <CardGrid>
+        <svelte:fragment slot="title">Payment estimates</svelte:fragment>
+        A breakdown of your estimated upcoming payment for the current billing period. Totals displayed
+        exclude accumulated credits and applicable taxes.
+        <svelte:fragment slot="aside">
+            <p class="text u-bold">
+                Due at: {toLocaleDate($organization?.billingNextInvoiceDate)}
+            </p>
+            <Card.Base variant="secondary" padding="s">
+                <Layout.Stack>
+                    <Layout.Stack direction="row" justifyContent="space-between">
+                        <Typography.Text color="--fgcolor-neutral-primary">
+                            {currentPlan.name} plan
+                        </Typography.Text>
+                        <Typography.Text>
+                            {isTrial || $organization?.billingPlan === BillingPlan.GITHUB_EDUCATION
+                                ? formatCurrency(0)
+                                : currentPlan
+                                  ? formatCurrency(currentPlan?.price)
+                                  : ''}
+                        </Typography.Text>
+                    </Layout.Stack>
+
+                    {#if currentPlan.budgeting && extraUsage > 0}
```

Comment:
this should be changed as addons can be projects or members,  and for new plan types it's projects, I used this method instead to show first the addons
https://github.com/appwrite/console/blob/f9ee8c2e0dc081ce55595f191496160b27b6adf5/src/routes/(console)/organization-%5Borganization%5D/billing/planSummary.svelte#L71-L72
and then for project breakdown, use project breakdown
https://github.com/appwrite/console/blob/f9ee8c2e0dc081ce55595f191496160b27b6adf5/src/routes/(console)/organization-%5Borganization%5D/billing/planSummary.svelte#L87-L88

I think you have reverted back these changes